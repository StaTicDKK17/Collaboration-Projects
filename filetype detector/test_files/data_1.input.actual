72
101
108
108
111
44
0
87
111
114
108
100
33
10
test_files/data_1.input: UTF-8 text
