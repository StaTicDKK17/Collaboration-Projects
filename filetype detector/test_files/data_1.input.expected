test_files/data_1.input: data
